src/components/Progress/Progress.tsx

```diff
@@ -144,7 +144,7 @@ export class Progress extends React.Component<ProgressProps> {
         const className = b('item', {theme: this.getTheme(), loading: this.props.loading});
 
         const offset = Progress.getOffset(value);
-        const style = {transform: `translateX(${offset}%)`};
+        const style = {insetInlineStart: `${offset}%`};
```

Comment:
Animating margin instead of transform will cause a reflow on update. Let's keep transform

Reply:
Yes, tnx. I changed to transform using css variables